Cartfile

```diff
@@ -0,0 +1 @@
+github "realm/realm-cocoa" ~> 2.0
```

Comment:
I've always been a bit _fuzzy_ (harhar) on the `~>` syntax. Does this restrict carthage from pulling the new `v3.x` of `realm-cocoa`? (i.e. does `~> 2.0` keep the dependency to the major version number?)

Reply:
Correct. This will allow FireKit to build its own binaries with the latest (at the time of building) version of Realm, while allowing dependent projects to select a version for themselves (so long as it's compatible with v2.x of Realm). And it will indeed restrict said clients and FireKit itself from pulling in 3.x: the ~> 2.0 specifier will allow any version between 2.0 up to but not including 3.0.